makefile

```diff
@@ -1,61 +0,0 @@
-# Binaries
-UGLIFY = ./node_modules/.bin/uglifyjs
-BROWSERIFY = ./node_modules/.bin/browserify
-KARMA = ./node_modules/karma/bin/karma
-JSHINT = ./node_modules/.bin/jshint
-DEREQUIRE = ./node_modules/.bin/derequire
-
-# Build debug and minified libraries
-all: \
-	dist/tangram.min.js \
-	dist/tangram.debug.js
-
-debug: dist/tangram.debug.js
-
-dist/tangram.debug.js: .npm $(shell ./build_deps.sh)
-	node build.js --debug=true --require './src/module.js' --runtime | $(DEREQUIRE) > dist/tangram.debug.js
-
-dist/tangram.min.js: dist/tangram.debug.js
-	$(UGLIFY) dist/tangram.debug.js -c warnings=false -m -o dist/tangram.min.js
-	@gzip dist/tangram.min.js -c | wc -c | awk '{ printf "%.0fk minified+gzipped\n", $$1 / 1024 }'
```

Comment:
Spitting out the gzipped size has kept me conscious of bundle size (vs. proactively checking it periodically), I think it's a nice thing to keep.